Description: Prefer first glob (in shared-mime-info database) for finding mimetype, not last
Author: Thomas Kluyver <takowl@gmail.com>
Origin: upstream, commit:f87e662ed27edf88152dd6980120b2cf64890db9

--- pyxdg-0.25.orig/xdg/Mime.py
+++ pyxdg-0.25/xdg/Mime.py
@@ -360,7 +360,8 @@ def _cache_database():
             if pattern.startswith('*.'):
                 rest = pattern[2:]
                 if not ('*' in rest or '[' in rest or '?' in rest):
-                    exts[rest] = mtype
+                    if rest not in exts:
+                        exts[rest] = mtype
                     continue
             if '*' in pattern or '[' in pattern or '?' in pattern:
                 globs.append((pattern, mtype))
